### DISCUSSION TITLE
what's the design style of the website, looks so good.

### DISCUSSION THREAD

**Discussion by BoneTM** [ariakit/ariakit#2401] [General]
is there any rule about the layout and color? is there a design system like the website, or it is author's self design?

**Comment by diegohaz** [comment]
It's a custom style defined in the tailwind.config.cjs file.

**Reply by BoneTM** [reply]
thanks, looks so good, gonna be a pro designer 👏

**Reply by FarooqAlaulddin** [reply]
I also notice that there is style.css with components preview in docs which is pretty awesome.
